src/db/query.js

```diff
@@ -63,7 +64,10 @@ async function query(query, db = null, removeNull = false, source = 'modal') {
 
           for (let j = 0; j < e.columns.length; j++) {
             const column = e.columns[j]
-
+            let fieldData = res[j]
+            if (PLATFORM === "WINDOWS" && typeof fieldData === "string") {
+              fieldData = fieldData.replace(/''/g, "'")// WTF!!!!
```

Comment:
Can we change this to single quotes as well `fieldData = fieldData.replace(/''/g, '\'') // WTF!!!!`